flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/windows.scala

```diff
@@ -150,7 +147,7 @@ class TumblingWindow(size: Expression) extends GroupWindow {
   def as(alias: String): TumblingWindow = as(ExpressionParser.parseExpression(alias))
 
   override private[flink] def toLogicalWindow: LogicalWindow =
-    ProcessingTimeTumblingGroupWindow(alias, size)
+    ProcessingTimeTumblingGroupWindow(name, size)
```

Comment:
Better to keep using "alias" here. By the way,
I found that the input of some functions use "name", like ProcessingTimeTumblingGroupWindow, while some others use "alias", like TumblingEventTimeWindow. Better to change them consistently to "alias". What do you think.